backend/app.py

```diff
@@ -21,6 +22,11 @@
 app = Flask(__name__)
 CORS(app)
 
+logging.basicConfig(filename="warnings",
+                    filemode='a',
+                    format='%(asctime)s,%(msecs)d %(name)s %(levelname)s %(message)s',
+                    datefmt='%H:%M:%S')
```

Comment:
Logging all errors to a file could be a good temp solution as long as we can still see this file for the deployed site.